app/models/pauri.rb

```diff
@@ -3,5 +3,5 @@
 class Pauri < ApplicationRecord
   belongs_to :chapter
   belongs_to :chhand
-  has_one :translation, :class_name => 'pauri_translation', :dependent => :destroy
+  has_one :translation, :class_name => 'PauriTranslation', :dependent => :destroy
```

Comment:
@anhadpreetsingh This was actually a mistake that we made earlier. This is another reason why the `pauri_spec` was failing :D 

I wanted you to figure it out hahaha, but in order for this migration to work, I need to add both:

`belongs_to :chhand` and fix the `class_name` 😄